alignak/objects/contactgroup.py

```diff
@@ -73,6 +73,8 @@ class Contactgroup(Itemgroup):
         'uuid':                 StringProp(default='', fill_brok=['full_status']),
         'contactgroup_name':    StringProp(fill_brok=['full_status']),
         'alias':                StringProp(fill_brok=['full_status']),
+        'contactgroup_members': ListProp(default=[], fill_brok=['full_status'],
```

Comment:
we should avoid this. Unless you are 100% sure this value is replaced or not use to append something. 
For example add_string_member function dos not ensure that (hasattr will return true) but add_string_unknown_member have this (bool([]) is false) 

See #403